DEVELOPER.md

```diff
@@ -0,0 +1,83 @@
+# Developing Beanstalk
+
+Development happens locally against a local forked version of the blockchain. We do not use testnets.
+
+### Repo Setup
+
+1. Clone the repo
+2. `yarn bootstrap`
+
+### Anvil: Forking Mainnet Locally
+
+We elect to use `anvil` instead of `hardhat` for local node forking as `anvil` is considerably faster than `hardhat` and properly caches the blockchain locally.
+
+- Install Foundry with `curl -L https://foundry.paradigm.xyz | bash` and reopen your terminal
+- Run `foundryup` to ensure you have downloaded the latest version
+- Start a locally forked node with the following command:
+
+```bash
+# optionally add
+# --fork-block-number <BLOCK_NUMBER>
+# if you want a specific point in time
+
+anvil --fork-url <FORK_RPC>  --chain-id 1337 --block-time 12
```

Comment:
We should include a note about the subgraph.

E.g.

"If you are using anvil with the local subgraph node, include the `--disable-block-gas-limit` option to avoid issues when indexing.